app/static/scss/admin/media/upload.scss

```diff
@@ -48,4 +48,16 @@
     .btn-choose {
         margin-right: 1rem;
     }
+    .error-message{
+        display: none;
+        background-color: $red;
+        color: $white;
+        height: 25px;
```

Comment:
Would you convert these `px` measurements to `rem`?  I think a height of `1.5rem` and padding of `0.25rem` would come pretty close to what you have now.